### PR TITLE
remove usage page check

### PR DESCRIPTION
**Pull request by jmarmstrong1207**
There is an issue with a user having a usage page of 0 rather than 1, causing errors for him. Not sure why, but removed these for a quick fix

### CODE REVIEW

**Comment by Codetector1374** [comment]
I understand the problem we have here, however, let's implement a temporary fix that we use the "highest interface found" for that device. Because on the C15 I have, it show up as three interface in recovery mode. And this will likely select the wrong interface for C15s.

**Comment by jmarmstrong1207** [comment]
I added what was checked before for c15 Is this fine?

**Comment by danil-tolkachev** [comment]
Won't work on my c18 without this PR.

File: src/annepro2.rs [hunk 1]
```diff
@@ -66,8 +63,8 @@ pub fn flash_firmware<R: std::io::Read>(target: AP2Target, base: u32, file: &mut
             }
             let dev = api.device_list().find(|dev| {
                 dev.vendor_id() == ANNEPRO2_VID &&
-                    ((dev.usage() == USAGE_C15 && dev.usage_page() == USAGE_PAGE_C15 && dev.product_id() == PID_C15) ||
-                     (dev.usage() == USAGE_C18 && dev.usage_page() == USAGE_PAGE_C18 && dev.product_id() == PID_C18))
+                    ((dev.usage() == USAGE_C15 && dev.product_id() == PID_C15 && dev.interface_number() == 1) ||
```
Comment (Codetector1374) [inline src/annepro2.rs]: If we are not checking `usage_page` then we should remove the check for `usage` as well.

**Review by Codetector1374 — CHANGES_REQUESTED** [review]

**Review by Codetector1374 — APPROVED** [review]
LGTM

**Pull request merged**